### PR TITLE
Colored text

### PR DESCRIPTION
**Pull request by UziTech**
I fixed colored brackets in comments/strings before a file is tokenized

and fixed colored text when entered before or after markers

Before this PR:

before

After:

after

### CODE REVIEW

File: lib/bracket-colorizer.js [hunk 1]
```diff
@@ -4,13 +4,12 @@ import { CompositeDisposable } from 'atom';
...
 
 export default {
-
   subscriptions: null,
 
   config: {
     brackets: {
       title: 'Brackets to match',
-      description: "The format should be, '{symbol_start};{symbol_end}'",
+      description: "The format should be, '{symbol_start};{symbol_end}'test",
```
Comment (vn-ki) [inline lib/bracket-colorizer.js]: What is the need for *test*?

Comment (UziTech) [reply lib/bracket-colorizer.js]: oops left over from debugging. I'll fix it

**Comment by vn-ki** [comment]
And... would you like to be a collaborator on this project? You obviously know more about atom than I do. I'm pretty sure I'll learn from you a lot.

**Comment by vn-ki** [comment]
> I can't guarantee I will have a lot of time to take on another project but I will help where I can.

That is totally okay. Anything you can contribute is great. :smile:

**Pull request merged**